### PR TITLE
Add rm cmd

### PR DESCRIPTION
**Pull request by Prnyself**

### CODE REVIEW

File: action/object.go [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+	if objectKey == "" {
+		return constants.ErrorQsPathObjectKeyRequired
+	}
+
+	_, err = contexts.SetupBuckets(bucketName, "")
+	if err != nil {
+		return
+	}
+	// Head to check whether object not found or forbidden
+	if _, err = helper.HeadObject(objectKey); err != nil {
```
Comment (Xuanwo) [inline action/object.go]: We should handle normal error and print user-friendly messages like:

404 -> object key not found
403 -> not enough permission

Comment (Prnyself) [reply action/object.go]: Roger that

File: cmd/mb.go [hunk 2]
```diff
@@ -27,5 +29,9 @@ bucket name should follow DNS name rule with:
 }
 
 func mbRun(_ *cobra.Command, args []string) (err error) {
+	// check zone flag (required)
+	if contexts.Zone == "" {
```
Comment (Xuanwo) [inline cmd/mb.go]: maybe we could check this in `ParseFlagIntoContexts`

We can get command name via `cmd.Name()`

File: main.go [hunk 3]
```diff
@@ -33,6 +33,7 @@ func init() {
 	application.AddCommand(cmd.StatCommand)
 	application.AddCommand(cmd.TeeCommand)
 	application.AddCommand(cmd.MbCommand)
```
Comment (Xuanwo) [inline main.go]: Let's keep commands added in alphabet order.

**Pull request closed without merging**